tests/encrypted-traffic-metadata/test.yaml

```diff
@@ -0,0 +1,20 @@
+requires:
+  features:
+    - HAVE_LIBJANSSON
+
+checks:
+  - filter:
+      # The number of records this filter should match.
+      count: 1
+      # The fields to match on.
+      match:
+        # Example match on event_type:
+        event_type: flow
+        # Example match on array item:
+        flow.etm.entropy: 7.945903301239014
```

Comment:
Hi. Thank you for this test. :)
I see that you've added a bunch of fields to the schema (https://github.com/OISF/suricata/pull/7415/files#diff-fe105e2d0f7fb994ce462f504d65673950ccbba1ad8c106899aa560ca591a0d2R1277), is it possible to have an event with all/most of those fields?
Multiple tests are also welcome.
You can use our createst script: https://github.com/OISF/suricata-verify/blob/master/createst.py to auto generate them.
Instructions on its usage are in the README of this project.

Reply:
I added 3 tests. One for https, one for udp and one for rdp. I also changed the tests to test for all new fields.